addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -0,0 +1,138 @@
+            this._createOrUpdateQuestion(createNext)
+        },
+
+        _cancelValidation: function () {
+            this.trigger_up('resetDisplay', {
+                $oldElem: this.$oldElem,
+                $elem: this.$el,
+                update: this.update
+            });
+        },
+
+        //--------------------------------------------------------------------------
+        // Private
+        //--------------------------------------------------------------------------
+
+        _createOrUpdateQuestion: function (createNext) {
+            var self = this;
+            var form = this.$('form');
+            if (this._isValidForm(form)) {
+                var values = this._serializeForm(form);
+                this._rpc({
+                    route: '/slides/slide/quiz/question_add_or_update',
+                    params: values
+                }).then(function (response) {
+                    console.log(response);
+                    if (response.update) {
+                        self.trigger_up('displayUpdatedQuestion', {
+                            element: self.$el,
+                            newQuestionData: response
+                        });
+                    } else {
+                        self.trigger_up('displayCreatedQuestion', {
+                            createNext: createNext,
+                            newQuestionData: response
+                        });
+                    }
+                }, function (error) {
+                    self._alertShow(form, error.message.data.arguments[0]);
+                });
+            } else {
+                this._alertShow(form, _t('Please fill up the question'));
+                this.$('.o_wslides_quiz_question input').focus();
+            }
+        },
+
+        _isValidForm: function(form) {
+            return form.find('.o_wslides_quiz_question input[type=text]').val().trim() !== "";
+        },
+
+        _serializeForm: function (form) {
+            var answers = [];
+            var sequence = 1;
+            form.find('.o_wslides_js_quiz_answer').each(function () {
+                var value = $(this).find('input[type=text]').val();
+                if (value.trim() !== "") {
+                    var answer = {
+                        'sequence': sequence++,
+                        'text_value': value,
+                        'is_correct': $(this).find('input[type=radio]').prop('checked') === true
+                    };
+                    answers.push(answer);
+                }
+            });
+            return {
+                'id': this.$el.data('id'),
+                'sequence': this.sequence,
+                'question': form.find('.o_wslides_quiz_question input[type=text]').val(),
+                'slide_id': this.slideId,
+                'answer_ids': answers
+            };
+        },
+
+        _alertShow: function (form, message) {
+            form.find('.o_wslides_js_quiz_create_error').removeClass('d-none');
```

Comment:
It's not "alerting" much.

Alert implies a popup window.
Maybe `displayFormError` ?